### PR TITLE
Change to point to Promscale

### PR DESCRIPTION
**Pull request by cevian**
The project timescale-prometheus was renamed to Promscale.
Adjust the name here and also bump version to beta.5

### CODE REVIEW

File: deploy/static/deploy.yaml [hunk 1]
```diff
@@ -5682,9 +5682,9 @@ spec:
 
     spec:
       containers:
-        - image: timescale/timescale-prometheus:0.1.0-alpha.3.1
+        - image: timescale/promscale:0.1.0-alpha.3.1
```
Comment (Harkishen-Singh) [inline deploy/static/deploy.yaml:5685]: just a comment: isn't the image lagging behind?

File: deploy/static/deploy.yaml [hunk 2]
```diff
@@ -6037,14 +6037,14 @@ spec:
...
 apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
-  name: tobs-timescale-prometheus-drop-chunk
+  name: tobs-promscale-drop-chunk
   labels:
-    app: tobs-timescale-prometheus
-    chart: timescale-prometheus-0.1.0-alpha.3.1
+    app: tobs-promscale
+    chart: promscale-0.1.0-alpha.3.1
```
Comment (Harkishen-Singh) [inline deploy/static/deploy.yaml:6047]: same goes for here

**Review by Harkishen-Singh — APPROVED** [review]
Apart from attention comments, lgtm.

File: deploy/static/deploy.yaml (hunk 1 above)
Comment (cevian) [reply deploy/static/deploy.yaml]: fixed

**Pull request merged**